Polygon.js

```diff
@@ -0,0 +1,167 @@
+	"label": "Polygon",
+	"creator": "czar",
+	"target": "^https?:\\/\\/(www.)?polygon.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2015-12-06 15:31:27"
+}
+
+/*
+    ***** BEGIN LICENSE BLOCK *****
+
+    Copyright © 2015 czar
+    http://en.wikipedia.org/wiki/User_talk:Czar
+
+    This file is part of Zotero.
+
+    Zotero is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Zotero is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+    ***** END LICENSE BLOCK *****
+*/
+
+function pageType (doc, url) { 
+	return ZU.xpathText(doc, '//script[contains(text(), "page_type :")]').split('page_type : \"')[1].split(/\"/)[0];
+}
+
+function detectWeb(doc, url) {
+	switch (pageType(doc, url)) { 
+		case "Article":
+			return "webpage";
+		case "HubPage":
+			return "multiple";
+	}
+	if (url.indexOf("/search?q=")!=-1){ return "multiple" }
+	return null;
+}
+
+function scrape(doc, url) {
+	var newItem = new Zotero.Item("webpage");
+	newItem.websiteTitle = "Polygon";
+	newItem.language = "en-US";
+
+	newItem.url = url;
+	newItem.title = ZU.xpathText(doc, '//h2[@class="m-entry__title"]');
+	newItem.date = ZU.xpathText(doc, '//meta[@property="article:published_time"]/@content').split("T")[0];
+
+	newItem.abstractNote = ZU.xpathText(doc, '//meta[@name="description"]/@content');
+	newItem.attachments.push({document:doc, title:"Polygon snapshot" ,mimeType:"text/html"});
+
+		// Authors
+	var authors = ZU.xpathText(doc, '//script[contains(text(), "dataLayer =")]').split('\"Author\":\"')[1].split(/\",/)[0].split(":");
+	do {
+		newItem.creators.push(ZU.cleanAuthor(authors[0], "author"));
+		authors.shift();
+	}
+	while (authors.length);
+	
+	newItem.complete();
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+        var items = {};
+        articles = [];
+        var titles = ZU.xpath(doc, '//div[@class="m-block__meta"]/h2/a')
+        for (var i = 0; i < titles.length; i++) {
+            items[titles[i].href] = titles[i].textContent;
+        }
+        Zotero.selectItems(items, function(items) {
+            if (!items) {
+                return true;
+            }
+            for (var i in items) {
+                articles.push(i);
+            }
+            Zotero.Utilities.processDocuments(articles, scrape);
+        })
+    }
+	scrape(doc, url);
```

Comment:
you want this to be 
```
else {
  scrape(doc, url)
}
```

otherwise you're also calling scrape directly on search results etc. which is causing the break you're seing.